remoteit/desktop
# [DESK-545] Secure login

### Description


Added ssl support to desktop for logging in remotely on a lan with https
**Note, I checked with mycal that it was ok to commit the certs, and he said it was fine because they could be seen by the user in the browser anyway**

### Checklist




### Test plan



1. Start up application on a device
2. Go to another device on the same network or use your same device
3. Go to http://[your device lan ip address]:29999
4. You should see a link at the bottom of the sign in page to change to https
5. Switch to https
6. Check all functionality still works

### Ticket



<https://remoteit.atlassian.net/browse/DESK-545>
